apps/design/frontend/src/api.ts

```diff
@@ -255,6 +255,7 @@ export const createDistrict = {
     return useMutation(apiClient.createDistrict, {
       async onSuccess(_, { electionId }) {
         await invalidateElectionQueries(queryClient, electionId);
+        await queryClient.refetchQueries(listDistricts.queryKey(electionId));
```

Comment:
Since the `listDistricts` query is no longer mounted on the Add District form, we have to refetch it during the mutation so that it's up to date with the new district before navigating back to the districts list (otherwise there's a lag before it shows up)